### ISSUE TITLE
Test in the adb-over-network environment

### ISSUE THREAD

**Issue by yyoon** [vanadium/madb#24]

**Comment by yyoon** [comment]
tl;dr
Seems to work fine. Closing for now.

---

Used three Nexus 9 devices for testing. Connected each device via USB cable one at a time, in order to enable the TCP mode for the device.

```
<for each device...>
$ adb tcpip 5555
```

Then, I disconnected all the USB data cables, and connected to these devices with their IP addresses. These devices were all on the same WiFi network, including the laptop on which I ran these adb commands.

```
<for each device...>
$ adb connect 192.168.86.xxx:5555
```

After this setup, the output of `adb devices` looks like this:

```
$ adb devices
List of devices attached
192.168.86.124:5555 device
192.168.86.123:5555 device
192.168.86.101:5555 device

```

I gave some nicknames (`WiFi01`~`WiFi03`) to these devices. The `<ip addr:port>` can be used in place of the serial number.

```
$ madb name set 192.168.86.101:5555 WiFi01
$ madb name set 192.168.86.123:5555 WiFi02
$ madb name set 192.168.86.124:5555 WiFi03
$ madb name list
Serial          Nickname
========================
192.168.86.101:5555 WiFi01
192.168.86.123:5555 WiFi02
192.168.86.124:5555 WiFi03
```

Finally, here's the output of running `madb start` command from one of our example projects.

```
[WiFi02]    372 KB/s (21315064 bytes in 55.930s)
[WiFi02]        pkg: /data/local/tmp/app-universal-debug.apk
[WiFi01]    360 KB/s (21315064 bytes in 57.805s)
[WiFi03]    357 KB/s (21315064 bytes in 58.173s)
[WiFi01]        pkg: /data/local/tmp/app-universal-debug.apk
[WiFi03]        pkg: /data/local/tmp/app-universal-debug.apk
[WiFi02]    Success
[WiFi02]    Stopping: io.v.android.apps.reader
[WiFi02]    Starting: Intent { cmp=io.v.android.apps.reader/.DeviceSetChooserActivity }
[WiFi03]    Success
[WiFi03]    Stopping: io.v.android.apps.reader
[WiFi03]    Starting: Intent { cmp=io.v.android.apps.reader/.DeviceSetChooserActivity }
[WiFi01]    Success
[WiFi01]    Stopping: io.v.android.apps.reader
[WiFi01]    Starting: Intent { cmp=io.v.android.apps.reader/.DeviceSetChooserActivity }
$ 
```
